### PR TITLE
betterdiscord-installer: 1.0.0-beta -> 1.1.1

### PR DESCRIPTION
**Pull request by 3nt3**
###### Motivation for this change
Newer version

###### Things done
Changed version, hash and path to desktop file

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/betterdiscord-installer/default.nix [hunk 1]
```diff
@@ -26,7 +26,7 @@ in appimageTools.wrapType2 {
     description = "Installer for BetterDiscord";
     homepage = "https://betterdiscord.app";
     license = licenses.mit;
-    maintainers = [ maintainers.ivar ];
+    maintainers = [ maintainers.ivar maintainers["3nt3"] ];
```
Comment (Krutonium) [inline pkgs/tools/misc/betterdiscord-installer/default.nix:29]: This has the wrong syntax, it should just be `maintainers.3nt3`

File: pkgs/tools/misc/betterdiscord-installer/default.nix [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 let
   pname = "betterdiscord-installer";
-  version = "1.0.0-beta";
+  version = "1.1.1";
   name = "${pname}-${version}";
 
   src = fetchurl {
     url = "https://github.com/BetterDiscord/Installer/releases/download/v${version}/Betterdiscord-Linux.AppImage";
-    sha256 = "103acb11qmvjmf6g9lgsfm5jyahfwfdqw0x9w6lmv1hzwbs26dsr";
+    sha256 = "1198fcbd1c966c47c26a0cbfc774131400ef13a67be9faa70d54506610f44b69";
```
Comment (Krutonium) [inline pkgs/tools/misc/betterdiscord-installer/default.nix:9]: Incorrect Hash, should be `0sabyh86cl2l1nkzmsbvlq9yy00l2dscggqcdb14fv4n3jyzr60i`

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -3533,6 +3533,12 @@
     github = "ericnorris";
     githubId = 1906605;
   };
+  "3nt3" = {
```
Comment (Krutonium) [inline maintainers/maintainer-list.nix:3536]: A friend of mine more familiar with this says this should likely be it's own PR.

**Review by Krutonium — CHANGES_REQUESTED** [review]
NGL I want this package updated too, so here's what I found as to why it hasn't been merged. And honestly, I'm disappointed that I'm the first person here to so much as review the PR *and* tell you where there's mistakes.

**Comment by Krutonium** [comment]
Additionally you should have @'d the existing maintainer as well - IvarWithoutBones

File: pkgs/tools/misc/betterdiscord-installer/default.nix [hunk 4]
```diff
@@ -1,12 +1,12 @@
 { appimageTools, lib, fetchurl }:
 let
   pname = "betterdiscord-installer";
-  version = "1.0.0-beta";
+  version = "1.1.1";
   name = "${pname}-${version}";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/betterdiscord-installer/default.nix:5]:
```suggestion
```
All app image tools now support pname+version.

**Comment by Quantenzitrone** [comment]
the current version is v1.5.3 

this PR is out of date

**Comment by Quantenzitrone** [comment]
i only started using Nix a few month ago

i need some time to get used to everything and understand how it is working

then i may contribute to the project too

**Comment by wegank** [comment]
Superseded by #280852.
